skimage/filters/thresholding.py

```diff
@@ -236,16 +236,66 @@ def threshold_local(image, block_size, method='gaussian', offset=0,
     return thresh_image - offset
 
 
-def threshold_otsu(image, nbins=256):
+def _validate_image_histogram(image, hist, nbins=None):
+    """Ensure that either image or hist were given, return valid histogram.
+
+    If hist is given, image is ignored.
```

Comment:
Could you please put this sentence in a `Notes` section?

Reply:
These should be in parameter descriptions.  "Must" is a good word to use when, well, you *must* provide the argument for it to work :)